features/keymap/key/automation-string.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Automation Key (String Macro)
+
+  Automation keys implement macro key behaviour.
+
+  For examples of this key in other smart keyboard firmware, see e.g.:
+
+  - [QMK's macro keys](https://docs.qmk.fm/feature_macros),
+
+  - [ZMK's macro behaviors](https://zmk.dev/docs/keymaps/behaviors/macros)
+
+  Example: string macro key
```

Comment:
It would be nice to add a test here; but, this'd be easier if the cucumber inputs supported `press_keymap_index` or so.